horizon/enforcer/api.py

```diff
@@ -553,6 +555,106 @@ async def is_allowed(
             )
         return result
 
+    @router.post(
```

Comment:
Remove

Reply:
why remove ?

Reply:
It was a redundant allowed that I duplicated.